pkgs/development/interpreters/wasmtime/default.nix

```diff
@@ -2,23 +2,27 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "wasmtime";
-  version = "0.21.0";
+  version = "0.32.1";
 
   src = fetchFromGitHub {
     owner = "bytecodealliance";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0q7wsnq5zdskxwzsxwm98jfnv2frnwca1dkhwndcn9yyz2gyw57m";
+    sha256 = "sha256-yWi4bz+DvTMy/Krt6uqg8M1Keah2ynEvgnupiBPahiY";
     fetchSubmodules = true;
   };
 
-  cargoSha256 = "1wlig9gls7s1k1swxwhl82vfga30bady8286livxc4y2zp0vb18w";
+  cargoSha256 = "sha256-IXrE1XVZUnyCbOcC0eEZKtI97YRA9Nap3u4tqcbEKf8";
 
   nativeBuildInputs = [ python cmake clang ];
   buildInputs = [ llvmPackages.libclang ] ++
    lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
   LIBCLANG_PATH = "${llvmPackages.libclang.lib}/lib";
 
+  configurePhase = ''
```

Comment:
This should likely be preConfigure. If you want to skip configurePhase use dontConfigure instead.